README.md

```diff
@@ -98,4 +98,28 @@ var config = konfiga(schema, { argv: myOwnParsedArgs });
 #### `env`
 By default, Konfiga get values from the enviroment using the `process.env` global. I'm not sure why you'd ever need to pass in your own equivilant object, but since I added the option for command line args, I felt I should add one for this too.
 
+#### `parsers`
+Konfiga comes with default parsers. To add more parsers, or override existing parsers, this array can be used. For example, to add a Set type:
+
+```js
+var config = konfiga(schema, {
+    parsers: [
```

Comment:
That was the original implementation. It looked a bit weird to me though. Open to changing it.